Resources/Prototypes/DeltaV/Entities/Objects/Fun/toys.yml

```diff
@@ -1,4 +1,4 @@
-- type: entity
+a- type: entity
```

Comment:
:trollface:

Reply:
WHAT DO YOU MEAN BY THIS qwq

Reply:
> WHAT DO YOU MEAN BY THIS qwq

He means you put an a before your dash. Instead of just an indent

Reply:
AGH!!!! FIXED!!! THANKS I AM VERY NEW TO CODING